dt-users/user-management.php

```diff
@@ -15,16 +15,16 @@ public static function instance() {
     public function __construct() {
         $url_path = dt_get_url_path();
         if ( self::has_permission() || self::non_admins_can_make_users() ) {
-            if ( strpos( $url_path, 'user-management/user' ) !== false || ( strpos( $url_path, 'user-management/add-user' ) !== false && ( current_user_can( "create_users" ) || self::non_admins_can_make_users() ) ) ){
+            if ( str_contains( $url_path, 'user-management/' ) && ( current_user_can( "create_users" ) || self::non_admins_can_make_users() ) ){
```

Comment:
@kodinkat is this part still needed?

Reply:
@corsacca I believe so; so as to future-proof against addition of any new page sections..

Reply:
@kodinkat the js part that we would need is no longer there.

Reply:
Oh. I mean for line 128

Reply:
and 24 i think